model/users_test.go

```diff
@@ -59,15 +59,18 @@ func TestCreateUser(t *testing.T) {
 		db, cleanUp := setupDBConnection(t, dbConfig)
 		defer cleanUp(t)
 
-		initialUsers := GetAllUsers(db)
+		initialUsers, err := GetAllUsers(db)
+		assert.Nil(t, err, "Expected no error when getting all users")
 
 		user := User{
 			GithubUsername: "testUsername",
 			GithubID:       123,
 		}
-		CreateUser(db, &user)
+		err = CreateUser(db, &user)
+		assert.Nil(t, err, "Expected no error when creating user")
```

Comment:
do we have cases for creating users with same username, getting non-exsiting users etc? But I guess we can finalise this more when our schema is more or less fixed. I can help with this as well.